docs/migration/dapp_migration.md

```diff
@@ -35,6 +35,7 @@ For starters, here's a sampling of exciting perks dapps with get with the Nitro
     - Auto-redeem will not be created if the user does not have enough balance to pay for `gasFeeCap * gasLimit` (meaning you can no longer set a max gas fee cap).
     - Deposited gas will be refunded to `excessFeeRefundAddress` if it cannot create an auto-redeem.
     - The user will be refunded the submission cost of their retryable if it is auto-redeemed.
+- **Arbitrum blockhash**: `blockhash(x)` returns Arbitrum blockhash of a specific L1 block `x` if `block.number - 256 <= x < block.number`, otherwise `0`. Arbitrum blockhash is a value deterministically generated from the L1 block number and state of the inbox. `blockhash(block.number)` will always return `0` in nitro.
```

Comment:
We should probably clarify the presence of fake L1 block hashes

Reply:
```suggestion
- **Arbitrum blockhash**: `blockhash(x)` returns Arbitrum blockhash of a specific L1 block `x` if `block.number - 256 <= x < block.number`, otherwise `0`. Arbitrum blockhash is **NOT** the L1 block hash but a value deterministically generated from the L1 block hash and state of the inbox. `blockhash(block.number)` will always return `0` in nitro in-line with ethereum.
```